integration/spark-common-cluster-test/src/test/scala/org/apache/spark/sql/common/util/QueryTest.scala

```diff
@@ -45,7 +45,7 @@ class QueryTest extends PlanTest with Suite {
   // Add Locale setting
   Locale.setDefault(Locale.US)
   CarbonProperties.getInstance()
-    .addProperty(CarbonCommonConstants.VALIDATE_DIRECT_QUERY_ON_DATAMAP, "false")
+    .addProperty(CarbonCommonConstants.SUPPORT_DIRECT_QUERY_ON_DATAMAP, "false")
```

Comment:
default value is false itself. no need to set again. remove it

Reply:
@ajantha-bhat Done